boa_inputgen/Cargo.toml

```diff
@@ -1,8 +1,32 @@
 [package]
 name = "boa_inputgen"
-authors = ["Addison Crump <me@addisoncrump.info>"]
-version = "0.1.0"
+version = "0.14.0"
 edition = "2021"
+rust-version = "1.58"
+authors = ["Addison Crump <me@addisoncrump.info>"]
+description = "Arbitrary JavaScript input generator used to fuzz Boa."
+repository = "https://github.com/boa-dev/boa"
+keywords = ["javascript", "js", "fuzzing", "testing"]
+categories = ["compilers"]
+license = "Unlicense/MIT"
+exclude = [
+    "../.vscode/*",
+    "../.editorconfig",
+    "../test262/*",
+    "../node_modules/*",
+    "../target/*",
+    "../dist/*",
+    "../.github/*",
+    "../assets/*",
+    "../docs/*",
+    "../*.js",
+    "../test_ignore.txt",
+    "../yarn.lock",
+    "../package.json",
+    "../index.html",
+    "../tests/*",
+    "../.github/*",
+]
```

Comment:
Actually, after the latest changes in #1952 all these excludes are not needed.

Reply:
Fixed.